docs/api-reference.md

```diff
@@ -690,6 +695,28 @@ Trace is the top level message of a trace.
 | AGGREGATION_FUNCTION_SUM | 5 |  |
 
 
+
+<a name="banyandb-model-v1-MeasureAggregate"></a>
+
+### MeasureAggregate
+
+
+| Name | Number | Description |
+| ---- | ------ | ----------- |
+| MEASURE_AGGREGATE_UNSPECIFIED | 0 |  |
```

Comment:
I think comments are required to generate docs here, to describe meanings of each function.

Reply:
Sure